packages/core/src/internal/share-latest.ts

```diff
@@ -91,7 +91,18 @@ const shareLatest = <T>(
     }
 
     if (!subscription) {
-      throw new Error("Missing subscription")
+      if (!outterSubscription) throw new Error("Missing Subscribe")
+
+      let err = EMPTY_VALUE
+      const sub = result.subscribe({
+        error(e) {
+          err = e
+        },
+      })
+      if (err !== EMPTY_VALUE) throw err
+
+      outterSubscription.add(sub)
+      return result.gV()
```

Comment:
I suspect if `source$` emits complete synchronously this could result in an infinite loop... I'll need to check

Reply:
Just checked, all good: this subscription doesn't emit the inner complete, so it doesn't unsubscribe, keeping the refCount > 0 and preventing the currentValue from being reset.

Reply:
No, but this is a good point, b/c before we go ahead and publish v1.0.0 I really want to revisit this decision of preventing the "enhanced" observable from completing...